ngraph/frontend/onnx_import/include/onnx_import/onnx.hpp

```diff
@@ -62,11 +62,15 @@ namespace ngraph
         /// \note       If stream parsing fails or the ONNX model contains unsupported ops,
         ///             the function throws an ngraph_error exception.
         ///
-        /// \param[in]  stream    The input stream (e.g. file stream, memory stream, etc).
+        /// \param[in]  stream     The input stream (e.g. file stream, memory stream, etc).
+        /// \param[in]  model_path The path to the imported onnx model.
+        ///                        It is required if the imported model uses data saved in external
+        ///                        files.
         ///
         /// \return     An nGraph function that represents a single output from the created graph.
         ONNX_IMPORTER_API
-        std::shared_ptr<Function> import_onnx_model(std::istream& stream);
+        std::shared_ptr<Function> import_onnx_model(std::istream& stream,
```

Comment:
@ilyachur As we discussed offline it is needed to establish how to pass `model_path` from [onnx_reader](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/readers/onnx_reader/ie_onnx_reader.cpp#L25)

Reply:
model path - is it a path to *.onnx file?

Reply:
Can we use this function if model doesn't require external files?

Reply:
@ilyachur Yes, it is path to *.onnx file. External files with weights have paths relative to onnx model path. The path is only used if the model use external files. So yes, we can use this function if model doesn't require external files.

Reply:
I just think how we can use it in the Inference Engine ReadNetwork. In this PR you didn't integrate this solution to IE API.

Reply:
@ilyachur I agree with you, but I think that is better to merge this PR which provides external data feature at the level of onnx importer. Providing this feature in Inference Engine ReadNetwork probably requires changes in API so I think that is better to do it separately (in this moment I have 2 propositions to achieve it).

Reply:
Do we needed support of external data in reader overload method which takes model as std::string [link](https://github.com/openvinotoolkit/openvino/blob/1afeb8470f8fb8fab3065258f933c7cf06a85fbd/inference-engine/src/inference_engine/ie_network_reader.hpp#L31) ?